examples/visualization/plot_study.py

```diff
@@ -86,3 +87,6 @@ def objective(trial):
 
     # Select parameters to visualize.
     plot_slice(study, params=["n_units_l0", "n_units_l1"]).show()
+
+    # Visualize plot importance
```

Comment:
Should it be "parameter" instead of "plot"? Oh, and a nit but let me make it plural and add a punctuation for consistency with the notebook/other comments in this file.
```suggestion
    # Visualize parameter importances.
```